R/rank.R

```diff
@@ -87,9 +86,7 @@ vec_rank <- function(x,
                      na_value = "largest",
                      nan_distinct = FALSE,
                      chr_transform = NULL) {
```

Comment:
It's still weird to me that we have this seemingly arbitrary `chr_transform` argument (why only chr? why doesn't the caller transform?) instead of something more suggestive that this is really about locales.